tests/backend/test_backends.py

```diff
@@ -26,17 +26,17 @@ def return_str_function():
 """
 
 
-@pytest.mark.skipif(bool(os.environ.get("TRAVIS", False)), reason="Venv backend doesn't work on Travis")
 @pytest.mark.parametrize(
-    ["requirements_location", "file_location"],
-    [
-        (None, None),
-        ("requirements/requirements.txt", None),
+    ["backend", "requirements_location", "file_location"], list(itertools.product(
+        (VenvBackend, DockerBackend),
+        (None, "requirements/requirements.txt"),
         (None, "test_package"),
-        ("requirements/requirements.txt", "test_package"),
-    ]
+    ))
 )
-def test_venv_backend(requirements_location, file_location):
+def test_backends(backend, requirements_location, file_location):
+    if os.environ.get("TRAVIS", False) and backend == VenvBackend:
+        raise pytest.skip("Venv Backend doesn't work on Travis")
```

Comment:
Should you raise this or call this? The call raises an exception.

Reply:
Yep, indeed, the call raises an exception so therefore the raise keyword is useless.